cli/lib/util.ts

```diff
@@ -160,7 +163,8 @@ function printNodeOptions (log = debug) {
   // returns string 'foo'
   ```
  */
-const dequote = (str) => {
+const dequote = (str: string): string => {
+  // @ts-expect-error
```

Comment:
@AtofStryker Can you add a comment for why this is needed

Reply:
added in f840a2b6722e154bc2aff8170a2fae5a2e7efc06. method exists but isn't typed